independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/CreateProjectHelper.java

```diff
@@ -78,10 +82,15 @@ public static void setJavaVersion(Map<String, Object> values, String javaTarget)
 
         Matcher matcher = JAVA_VERSION_PATTERN
                 .matcher(javaTarget != null ? javaTarget : System.getProperty("java.version", ""));
-        if (matcher.matches() && Integer.parseInt(matcher.group(1)) < 11) {
-            values.put(ProjectGenerator.JAVA_TARGET, "8");
+
+        if (matcher.matches()) {
+            String versionExtracted = matcher.group(1);
+            String version = JAVA_VERSIONS_LTS.stream()
+                    .filter(e -> e.equals(versionExtracted))
+                    .findFirst().orElse(DEFAULT_JAVA_VERSION);
```

Comment:
```suggestion
            String version = JAVA_VERSIONS_LTS.contains(versionExtracted) ? versionExtracted : DEFAULT_JAVA_VERSION;
```

Reply:
Yes, using contains is more efficient. Thanks @gastaldi

Reply:
I also added a suggestion to have a `Set`

Reply:
@gastaldi @ia3andy, done!!